src/analysis/typedtrie.ml

```diff
@@ -380,6 +384,7 @@ let rec build ~local_buffer ~trie browses : t =
           | `Sg  sg ->
             let sg = lazy (build ~local_buffer ~trie [of_signature sg]) in
             f (Included (Items sg))
+          | `Hole -> assert false
```

Comment:
Is that right? I haven't really thought about it, but can't I `include _`? If indeed I can't, perhaps put a comment in there explaining why the assert is correct?